src/main/java/org/apache/sysml/runtime/instructions/cp/AggregateBinaryCPInstruction.java

```diff
@@ -34,7 +34,8 @@
 
 public class AggregateBinaryCPInstruction extends BinaryCPInstruction {
 
-	private AggregateBinaryCPInstruction(Operator op, CPOperand in1, CPOperand in2, CPOperand out, String opcode, String istr) {
+	public AggregateBinaryCPInstruction(Operator op, CPOperand in1, CPOperand in2, CPOperand out, String opcode,
+			String istr) {
```

Comment:
Please don't introduce such a corrupted formatting (only break the line if there are sufficiently many inputs and try to break lines evenly).